### ISSUE TITLE
Media queries break pseudo classes

### ISSUE THREAD

**Issue by ayastreb** [Andarist/stylis-plugin-extra-scope#9]
If there's at least one media-query in the style, it's breaking pseudo and nested selectors.
See reproduction here:
https://codesandbox.io/s/increase-css-specificity-forked-lum7o

Notice that `:hover` selector doesn't work, because CSS generated for it is:
```
#extra-scope #extra-scope .cssss-161048u:hover{background-color:blue;}
```
the scope is added twice and therefor CSS is not applied

**Comment by ayastreb** [comment]
hi @Andarist! did you have a chance to look at this issue/MR? Would it be possible to merge/publish the fix?

**Comment by Andarist** [comment]
Thanks for a ping - I've found some problems with your proposed fix. I can't say I fully understand the original issue nor the extra issue that I have found but after some thinking, I've found a way to simplify the implementation *a lot* (by preparing the scoped selectors once, in the beginning, instead of trying to fix things for all generated rules).

I hope to release a new version with this tomorrow.
